src/web/views/jabiru.jade

```diff
@@ -1,21 +1,21 @@
 doctype html
-head
-    script(src='/assets/js/jquery-2.1.3.min.js')
-    script(src='/assets/js/jabiru.js')
-    link(rel='stylesheet', type='text/css', href='/assets/css/jabiru.css')
 html
-    include ./layouts/head.jade
-body
-    include ./layouts/topbar.jade
-
-    .image
-      img#page-image(src='/images/0')
-    .text
-      textarea#page-text.textBox(cols='')
-    .buttons
-      a#finish-btn.button(href="/outputPreview")= _('op_finish')
-      #save-btn.button: p:= _('j_save')
-      #next-btn.button: p:= _('s_next')
-      #back-btn.button: p:= _('np_new_book_go_back')
-
-    include ./layouts/footer.jade
+    head
+        script(src='assets/js/jquery-2.1.3.min.js')
```

Comment:
This include is not required is already included in https://github.com/LabExperimental-SIUA/LibreScan/blob/a845629987179d08cb15c2548a236d1bcbd862cf/src/web/views/layouts/footer.jade#L7